### ISSUE TITLE
export_to_video crashe after opencv version upgrade

### ISSUE THREAD

**Issue by soulteary** [huggingface/diffusers#6476]
### Describe the bug

Due to some issues with OpenCV dependencies, even if you use the latest version of Gradio, Diffusers, and Transformers, you will still get an error when saving the video.

Maybe later, submit a PR

### Reproduction

Simply put, when loading the model and processing user input, using export_to_video will trigger a program crash.

```poc
pipe = StableVideoDiffusionPipeline.from_pretrained("/app/stabilityai/stable-video-diffusion-img2vid-xt", torch_dtype=torch.float16, variant="fp16")
pipe.to("cuda")
pipe.unet = torch.compile(pipe.unet, mode="reduce-overhead", fullgraph=True)
pipe.vae = torch.compile(pipe.vae, mode="reduce-overhead", fullgraph=True)

...

frames = pipe(image, decode_chunk_size=decoding_t, generator=generator, motion_bucket_id=motion_bucket_id, noise_aug_strength=0.1, num_frames=25).frames[0]
export_to_video(frames, video_path, fps=fps_id)
```

### Logs

```shell
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/gradio/queueing.py", line 489, in call_prediction
    output = await route_utils.call_process_api(
  File "/usr/local/lib/python3.10/dist-packages/gradio/route_utils.py", line 232, in call_process_api
    output = await app.get_blocks().process_api(
  File "/usr/local/lib/python3.10/dist-packages/gradio/blocks.py", line 1561, in process_api
    result = await self.call_function(
  File "/usr/local/lib/python3.10/dist-packages/gradio/blocks.py", line 1179, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "/usr/local/lib/python3.10/dist-packages/anyio/to_thread.py", line 56, in run_sync
    return await get_async_backend().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.10/dist-packages/anyio/_backends/_asyncio.py", line 2134, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.10/dist-packages/anyio/_backends/_asyncio.py", line 851, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.10/dist-packages/gradio/utils.py", line 678, in wrapper
    response = f(*args, **kwargs)
  File "/app/web/app.py", line 94, in hey
    video, seed = sample(img, seed, randomize_seed, motion_bucket_id, fps_id)
  File "/app/web/app.py", line 55, in sample
    export_to_video(frames, video_path, fps=fps_id)
  File "/usr/local/lib/python3.10/dist-packages/diffusers/utils/export_utils.py", line 122, in export_to_video
    import cv2
  File "/usr/local/lib/python3.10/dist-packages/cv2/__init__.py", line 181, in <module>
    bootstrap()
  File "/usr/local/lib/python3.10/dist-packages/cv2/__init__.py", line 175, in bootstrap
    if __load_extra_py_code_for_module("cv2", submodule, DEBUG):
  File "/usr/local/lib/python3.10/dist-packages/cv2/__init__.py", line 28, in __load_extra_py_code_for_module
    py_module = importlib.import_module(module_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/usr/local/lib/python3.10/dist-packages/cv2/typing/__init__.py", line 168, in <module>
    LayerId = cv2.dnn.DictValue
AttributeError: module 'cv2.dnn' has no attribute 'DictValue'
```

### System Info

docker env: nvcr.io/nvidia/pytorch:23.12-py3

pip install transformers==4.35.2
pip install accelerate==0.25.0
pip install diffusers 0.25.0
pip install gradio==4.13.0
pip install opencv-python==4.9.0.80

### Who can help?

@yiyixuxu

**Comment by soulteary** [comment]
The cause of the problem comes from here, https://github.com/opencv/opencv-python/issues/884

i've made a simple tool to fix the problem, hope it helps.

```bash
pip install opencv-fixer==0.2.5
python -c "from opencv_fixer import AutoFix; AutoFix()"
```

**Comment by patrickvonplaten** [comment]
I think this needs to be fixed via: https://github.com/opencv/opencv-python/issues/884 no?

**Comment by soulteary** [comment]
> I think this needs to be fixed via: [opencv/opencv-python#884](https://github.com/opencv/opencv-python/issues/884) no?

maybe, or fix by docker env provider, nvidia, lol :D
